gluten-ut/common/src/test/scala/org/apache/spark/sql/GlutenTestsTrait.scala

```diff
@@ -293,7 +293,7 @@ trait GlutenTestsTrait extends GlutenTestsCommonTrait {
 
   def shouldNotFallback(): Unit = {
     TestStats.offloadGluten = false
-    if (BackendsApiManager.getBackendName != GlutenConfig.GLUTEN_CLICKHOUSE_BACKEND) {
+    if (BackendTestUtils.isCHBackendLoaded()) {
```

Comment:
if (!BackendTestUtils.isCHBackendLoaded()) ?

Reply:
Thanks for catching!